test/records/test_models.py

```diff
@@ -65,98 +219,44 @@ def test_empty_for_optional_attributes(self):
         self.assertEqual(self.record.is_tna, False)
         self.assertEqual(self.record.is_digitised, False)
 
-    def test_iaid(self):
-
-        self.record = Record(self.source)
-
-        # patch raw data
-        self.record._raw["@template"]["details"]["iaid"] = "C123456"
-
+    def test_property_iaid(self):
```

Comment:
```suggestion
    def test_iaid(self):
```
`test_property_` for individual record methods/functions/"attributes" feels like an overload of a naming convention to follow, then there should be some kind of documentation too. Also what if we have an api attribute has the name `def property` so this could become `test_property_property`!
I think the original naming is good enough.